code/game/objects/items/grenades/flashbang.dm

```diff
@@ -26,12 +26,19 @@
 
 //Flash
 	if(M.flash_act(affect_silicon = 1))
-		M.Paralyze(max(200/max(1,distance), 60))
+		if(M.flash_act())
+			M.Paralyze(max(20/max(1,distance), 5))
+			M.Knockdown(max(200/max(1,distance), 60))
+		else
+			M.Paralyze(max(200/max(1,distance), 60))
+
 //Bang
 	if(!distance || loc == M || loc == M.loc)	//Stop allahu akbarring rooms with this.
-		M.Paralyze(200)
+		M.Paralyze(20)

```

Comment:
No. If you are on top of a grenade without protection, you get the exact same (pretty much) as what you get if you are on top of it with protection. That is what the code is intended to do. Charging in with a grenade in your hand will not help you in any way, since chances are your victim will get up before you.

Reply:
It will help a lot if you are a sec officer, with proper eye protection, against targets without. Because your targets will have up to 200 Knockdown in the current version of your PR, while you'll have 100.

I main sec officer, and flashbangs are super powerful. They are a very handy tool. I understand nerfing them, but I don't understand why buffing suicide-bombing tactics, and why removing the need to cook the nade to properly use it.

Reply:
Oh I actually didn't notice that knockdown is 100 imma go change that to 200 sorry.

Reply:
I literally... ok